drivers/staging/vc04_services/vchiq-mmal/mmal-vchiq.c

```diff
@@ -44,6 +44,12 @@ MODULE_VERSION("0.0.1");
  */
 #define VCHIQ_MMAL_MAX_COMPONENTS 64
 
+/*
+ * timeout for synchronous msg responses in seconds.
```

Comment:
Capitalise `Timeout`, since it starts a sentence? I only noticed because of the line below.